tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -10,34 +10,105 @@ const {
   sendEventAndCheckResponse,
   cleanMockTestFixture,
   checkInClusterEventDelivery,
+  waitForSubscriptionsTillReady,
+  setEventMeshSourceNamespace,
 } = require("../test/fixtures/commerce-mock");
+const {
+  switchEventingBackend,
+  createEventingBackendK8sSecret,
+  deleteEventingBackendK8sSecret,
+} = require("../utils");
 
 describe("Eventing tests", function () {
   this.timeout(10 * 60 * 1000);
   this.slow(5000);
   const testNamespace = "test";
   const testStartTimestamp = new Date().toISOString();
+  const backendK8sSecretName = process.env.BACKEND_SECRET_NAME || "eventing-backend";
+  const backendK8sSecretNamespace = process.env.BACKEND_SECRET_NAMESPACE || "default";
+  const eventMeshSecretFilePath = process.env.EVENTMESH_SECRET_FILE || "";
+
+  before(async function() {
+    // runs once before the first test in this block
+
+    // If eventMeshSecretFilePath is specified then create a k8s secret for eventing-backend
+    // else use existing k8s secret as specified in backendK8sSecretName & backendK8sSecretNamespace
+    if (eventMeshSecretFilePath != "") {
```

Comment:
My IDE seems to prefer !== since it will prevent automatic type conversion. Not sure if it is important here!

Reply:
updated to !==

Reply:
If that is the case, should we do it for all of them!?

Reply:
also updated the other occurance.